docs/user/en/reference/Rendering Preferences.rst

```diff
@@ -50,17 +50,39 @@ Other Preferences
 
 .. _project_preferences-hide-renderer-job:
 
-HIDE_RENDER_JOB
-~~~~~~~~~~~~~~~
-Allows to control, if executed jobs are shown in Progress View. If this option is
-available or set to **true**, the jobs are not shown. Otherwise, which is the default, jobs
-are shown in Progress View. This option is not intended to be set or changed at runtime.
-Use *.option* file in installation folder to configure application behavior:
+Hide render jobs in Progress View
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The option **HIDE_RENDER_JOB** allows to control, whether executed jobs are shown in Progress View.
+If this option is available and set to **true**, the jobs are not shown. Otherwise, which is the
+default, jobs are shown in Progress View.
+
+This option is not intended to be set or changed at runtime.
+Use *.options* file in installation folder and include this file for instance with the Eclipse
+command line option -pluginCustomization to configure application behavior:
```

Comment:
Nice, works like a charm! And I'm wondering if we can enable this option for uDig Application per default.